freud/order.pyx

```diff
@@ -175,10 +175,24 @@ cdef class Nematic(_Compute):
     def compute(self, orientations):
         R"""Calculates the per-particle and global order parameter.
 
+        Example::
+
+            >>> # Assuming user has set director = np.array[x,y,z]
```

Comment:
The synatx for `np.array[x, y, z]` should be `np.array([x, y, z])` -- but judging from the examples for the `Nematic`  order parameter, you can just use a list like `[1, 0, 0]` for the example. 

I think it would be best, though, for this example, to remove this comment and copy the value from the example for the `director` variable below.